scripts/cmd_args.py

```diff
@@ -256,6 +284,14 @@ def add_eval_options(parser):
     """
     add_inference_options(parser)
 
+    parser.add('--cleanbw', type=type_or_none_fn(str),
```

Comment:
Why are cleanbw and cleanpeakfile present under both common options and inference options?

Reply:
cleanbw and cleanpeakfile are relevant for training and evaluation. This is the evaulation options, not inference :) 

I removed it from common options.